treeherder/etl/perf.py

```diff
@@ -177,7 +191,8 @@ def _load_perf_artifact(project_name, reference_data, job_data, job_guid,
                 signature=signature,
                 push_timestamp=push_timestamp,
                 defaults={'value': suite['value']})
-            if datum_created and (not is_try_repository):
+            if (signature.should_alert is not False and datum_created and
```

Comment:
Likewise, "is not False" is meant as a check that the submitter didn't *express* an intention that we shouldn't alert on the series (in which case we should fall to the default behavior of only alerting on subtests without a parent signature)